### ISSUE TITLE
Teams?

### ISSUE THREAD

**Issue by cirodalex** [stoopkid1/ezleague#134]
Hi, thank you for this great work :)
I want to use your app to manage FIFA league and FIFA tournaments.
Installation and configuration it's very easy, I Have only an error in \admin\header.php on line 31 regarding date function but doesn't matter now.
I see the teams was suspended, it's because you're working on it? I also see in my db there is no ezlteams table.
I hope you will give me notice about it :+1:

**Comment by zigstum** [comment]
The error goes away once you have entered your current timezone in the admin area.
The teams are not suspended (as far as I can tell), that little red box icon is just to show you what it looks like when a team is suspended.
I think the table you are expecting to see is called ezlguilds.
I only found this yesterday and have only spent five mins looking at it, but I hope that helps.

**Comment by cirodalex** [comment]
Ok, but in the admin->teams there are no php pages to create, edit or view
single team
Il 11/mar/2015 13:06 "zigstum" <notifications@github.com> ha scritto:

**Comment by zigstum** [comment]
Teams are created by users, not by the admin in the admin control area.

**Comment by cirodalex** [comment]
Ok, so I will try like user.
Thank you ;)
Il 11/mar/2015 14:39 "zigstum" <notifications@github.com> ha scritto:

**Comment by nmetin** [comment]
hello, 

I am testing out this script on my host. 
I have the same problem in header.php with the timezone function. 
I tried to change the timezone but I can't access this page.

I can access admin/settings.php?page=site
But admin/settings.php?page=site#site-timezone doesn't load 

I did bypass the problem by removing some code out of header.php.
But now i am having problems elsewhere. 

So how can I fix this problem the proper way?
